phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -25,12 +25,9 @@
         </whitelist>
     </filter>
 
-    <!-- Run `composer require symfony/phpunit-bridge` before enabling this extension -->
```

Comment:
This removes any diff between phpunit.xml.dist shipped by simple-phpunit vs phpunit recipes.
Without this change, what ppl get is random, leading to the deprecation report being unavailable.